tsc_list.txt

```diff
@@ -37,94 +37,94 @@ Format:
-<MYB	1	d---	MY Bump	Causes the PC to hop in the direction opposite of W. Using up or down causes the jump to be vertical.	0	0	1	4
-<MYD	1	d---	MY Direction	Causes the PC to face direction W.	0	0	1	4
-<NCJ	2	ne--	Npc Check Jump	Jump to event X if any entity of type W exists.	0	0	1	4
-<NOD	0	----	NOD	Wait for player input before resuming script.	0	0	1	4
-<NUM	1	#---	NUMber	Prints the value [4a5b34+W*4] to the message box. Use 0000 to print the last used W from compatible commands (eg AM+).	0	0	1	4.
-<PRI	0	----	PRevent Interaction	Lock player controls and freeze game action.	0	0	1	4
-<PS+	2	#m--	Portal Slot +	Set teleporter slot W to event X. Selecting slot W while using the teleporter menu will jump to event X.	0	0	1	4
-<QUA	1	.---	QUAke	Shake the screen for W ticks.	0	0	1	4
-<RMU	0	----	Restore MUsic	Resume the song last played.	0	0	1	4
-<SAT	0	----	Speed-up All Text	Instantly display text. Use before a <MSG/2/3; works until <END. Same command as <CAT.	0	0	1	4
-<SIL	1	l---	Show ILlustration	Show illustration W (during credits).	0	0	1	4
-<SK+	1	F---	SKipflag +	Set skipflag W.	0	0	1	4
-<SK-	1	F---	Skipflag -	Clear skipflag W.	0	0	1	4
-<SKJ	2	Fe--	SKipflag Jump	Jump to event X if skipflag W is set.	0	0	1	4
-<SLP	0	----	Show Location Portals	Show the teleporter menu.	1	0	1	4
-<SMC	0	----	Show My Character	Unhides the PC.	0	0	1	4
-<SMP	2	xy--	Shift MaP	tile Subtract 1 from the tile type at coordinates W:X. Does not create smoke.	0	0	1	4
-<SNP	4	nxyd	Set NPc	Create an entity of type W at coordinates X:Y with direction Z.	0	0	1	4
-<SOU	1	s---	SOUnd	Play sound effect W.	0	0	1	4
-<SPS	0	----	Start Propeller Sound	Start the propeller sound.	0	0	1	4
-<SSS	1	#---	Start Stream Sound	Start the stream sound with volume W.	0	0	1	4
-<STC	0	----	Save Time Counter	Saves current time to 290.rec.	0	0	1	4
-<SVP	0	----	SaVe Profile	Saves current game.	0	0	1	4
-<TAM	3	aaA-	Trade ArMs	Trade weapon W for weapon X and set max ammo to Y. Use 0000 to keep the same amount of ammo.	0	0	1	4
-<TRA	4	mexy	TRAnsport	Travel to map W, run event X, and move the PC to coordinates Y:Z.	1	0	1	4
-<TUR	0	----	Text UnRead?	Instantly display text. Use after a <MSG/2/3; works until another <MSG/2/3 or an <END.	0	0	1	4
-<UNI	1	#---	UNIverse?	Set character movement type. Use 0000 for normal, 0001 for zero-G and 0002 to disallow movement.	0	0	1	4
-<UNJ	2	#e--	UNiverse Jump	Jump to event X if movement is of type W (0000 for normal, 0001 for zero-G).	0	0	1	4
-<WAI	1	.---	WAIt	Pause script for W ticks.	0	0	1	4
-<WAS	0	----	WAit until Standing	Pause script until character is on ground.	0	0	1	4
-<XX1	1	l---	XX1	Show the island falling in manner W. Use 0000 to have it crash and 0001 to have it stop midway.	0	0	1	4
-<YNJ	1	e---	Yes/No Jump	Prompt Yes/No; jump to event W if No is selected.	0	0	1	4
-<ZAM	0	----	Zero ArMs	Sets all weapon energy to zero.	0	0	1	4
\ No newline at end of file
+<AE+	0	----	Arms Energy +	Refill all weapon ammo.	0	0	1	4	4	4	4
```

Comment:
this file is getting kind of wild.. I guess it serves its purpose but maybe at some point it would be nice if there was an easier way to maintain it.

Reply:
I'm working on a TSC list editor.